lib/modules/apostrophe-browser-utils/public/js/lean.js

```diff
@@ -27,13 +27,14 @@
 
   apos.utils.emit = function(el, name, data) {
     var event;
-    if (!window.CustomEvent) {
-      // bc
+    try {
```

Comment:
🤯 TIL `try...catch` isn't only for Promises. I guess that makes sense!

Reply:
Hee! This is probably a common impression among JavaScript developers because we got "it's useless because async" beaten into our heads for so long. Now it's back because async functions allow it wrapped around `await`, but folks still forget it is perfectly fine for synchronous code too (where it was born!)